docs/product/run.mdx

```diff
@@ -26,4 +26,15 @@ Nhost Run offers several key advantages for running workloads alongside your pro
 
 By leveraging Nhost Run, you can optimize the performance, reliability, and cost-efficiency of your project by running workloads alongside it.
 
-
+## Guides
+
+<CardGroup cols={4}>
+  <Card title="Getting Started" icon="square-1" href="../guides/run/getting-started">
+  </Card>
+  <Card title="Configuration" icon="square-2" href="../guides/run/configuration">
+  </Card>
+  <Card title="Resources" icon="square-4" href="../guides/run/resources">
+  </Card>
+  <Card title="CLI & CI Deployments" icon="square-3" href="../guides/run/cli-deployments">
```

Comment:
aren't we missing the new healthcecks page?

Reply:
I hand picked some pages but I could add all of them if that's better.

Reply:
yeah, let's do all for now

Reply:
done